fbpic/openpmd_diag/inputdeck_diag.py

```diff
@@ -0,0 +1,120 @@
+"""
+
+inputdeck_diag.py
+
+opmd derived diagnostic for saving the input deck
+into each simulation dump.
+
+"""
```

Comment:
Could you replace the header by the following one:
```
# Copyright 2019, FBPIC contributors
# Authors: Kris Poder
# License: 3-Clause-BSD-LBNL
"""
This file defines the class InputScriptDiagnostic.

It allows saving the input script into each simulation dump.
"""

Reply:
Sorted!